### PR TITLE
Use http helper2

### PR DESCRIPTION
**Pull request by ganmacs**
Retry https://github.com/fluent/fluent-plugin-prometheus/pull/136/

### CODE REVIEW

File: lib/fluent/plugin/in_prometheus.rb [hunk 1]
```diff
@@ -60,49 +62,106 @@ def multi_workers_ready?
...
+        SSLEnable: true
+      }
+
+      if @ssl['certificate_path']
+        cert = OpenSSL::X509::Certificate.new(File.read(@ssl['certificate_path']))
+        ssl_config[:SSLCertificate] = cert
+      end
+
+      if @ssl['private_key_path']
+        key = OpenSSL::PKey::RSA.new(File.read(@ssl['private_key_path']))
```
Comment (cosmo0920) [inline lib/fluent/plugin/in_prometheus.rb:143]: nit: `OpenSSL::PKey.read`?

https://ruby-doc.org/stdlib-2.4.0/libdoc/openssl/rdoc/OpenSSL/PKey.html#method-c-read

File: lib/fluent/plugin/in_prometheus.rb [hunk 2]
```diff
@@ -111,16 +170,8 @@ def start
       end
 
       thread_create(:in_prometheus) do
```
Comment (repeatedly) [inline lib/fluent/plugin/in_prometheus.rb:172]: `in_prometheus_webrick` is better

File: lib/fluent/plugin/in_prometheus.rb (hunk 1 above)
Comment (ganmacs) [reply lib/fluent/plugin/in_prometheus.rb]: fixed https://github.com/fluent/fluent-plugin-prometheus/pull/148/commits/bc7c2ba3c47e8289f4a726c1b94b5dd444df3587

File: lib/fluent/plugin/in_prometheus.rb (hunk 2 above)
Comment (ganmacs) [reply lib/fluent/plugin/in_prometheus.rb]: right. https://github.com/fluent/fluent-plugin-prometheus/pull/148/commits/49aa9c25b18384102dbb8e737fd7e9c165053cd7

**Pull request merged**